modules/training/ajax/getTabContent.php

```diff
@@ -66,6 +66,21 @@
         $instrument . '.mp4',
         $type
     );
+} else if ($tabInformation['TrainingType'] == 'vimeo') {
+    $oembed_endpoint = 'http://vimeo.com/api/oembed';
+    $video_url       = 'http://vimeo.com/' . $tabInformation['Content'];
```

Comment:
These are http, but the cross-origin policy is https (@johnsaigle's comment from the meeting..)